static/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.tsx

```diff
@@ -158,11 +163,7 @@ export default class SentryApplicationDetails extends AsyncView<Props, State> {
     const {orgId} = this.props.params;
     const baseUrl = `/settings/${orgId}/developer-settings/`;
     const url = app ? baseUrl : `${baseUrl}${data.slug}/`;
-    if (app) {
-      addSuccessMessage(t('%s successfully saved.', data.name));
-    } else {
-      addSuccessMessage(t('%s successfully created.', data.name));
-    }
+    addSuccessMessage(t('%s successfully %s.', data.name, app ? 'saved' : 'created'));
```

Comment:
Unfortunately doing this means the string `saved` and `created` will be no longer transalted.